RLTest/redis_cluster.py

```diff
@@ -44,6 +44,18 @@ def printEnvData(self, prefix=''):
             print(Colors.Yellow(prefix + 'shard: %d' % (i + 1)))
             shard.printEnvData(prefix + '\t')
 
+    def getInformationBeforeDispose(self):
+        shards = []
+        for shard in self.shards:
+            shards.append(shard.getInformationBeforeDispose())
+        return shards
+
+    def getInformationAfterDispose(self):
+        shards = []
+        for shard in self.shards:
```

Comment:
List comprehension

return [s.getInformationAfterDispose() for s in self.shards]